VideoApp/VideoApp/NewCode/Views/Participant/ParticipantView.swift

```diff
@@ -31,12 +31,20 @@ struct ParticipantView: View {
                 .padding()
 
             if viewModel.cameraTrack != nil {
-                /// Use opacity to hide the view when the server switches the track off due to bandwidth constraints. This will keep
+                /// Use opacity to hide the video when the server switches the track off due to bandwidth constraints. This will keep
                 /// the view in the hierarchy which will signal to the server that the UI wants to render this video. The server will
-                /// switch the track on when bandwidth constraints allow. If the view was completely removed from the hierarchy
+                /// switch the track on when bandwidth constraints allow. If the video was completely removed from the hierarchy
                 /// the server would never switch the track on.
-                SwiftUIVideoView(videoTrack: $viewModel.cameraTrack, shouldMirror: $viewModel.shouldMirrorCameraVideo)
-                    .opacity(viewModel.isCameraTrackSwitchedOff ? 0 : 1)
+                ZStack {
+                    Color.black // For black bars
+
+                    SwiftUIVideoView(
+                        videoTrack: $viewModel.cameraTrack,
+                        shouldMirror: $viewModel.shouldMirrorCameraVideo,
+                        fill: viewModel.shouldFillCameraVideo
```

Comment:
I think I could probably create a nested struct on this view model just for camera stuff and then probably have one parameter here and better semantics. But didn't want to do that refactor in this PR.